composer.json

```diff
@@ -17,20 +17,13 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
-        "zendframework/zend-form": "~2.3",
-        "zendframework/zend-http": "~2.3",
-        "zendframework/zend-json": "~2.3",
-        "zendframework/zend-mvc": "~2.3",
-        "zendframework/zend-i18n": "~2.3",
-        "zendframework/zend-inputfilter": "~2.3",
-        "zendframework/zend-servicemanager": "~2.3",
-        "zendframework/zend-validator": "~2.3",
-        "zendframework/zend-view": "~2.3"
+        "php": ">=5.6 || >= 7.0",
+        "zendframework/zendframework": "2.*"
```

Comment:
Please don't depend on the complete zendframework, but only on the seperate packages.